pvlib/irradiance.py

```diff
@@ -323,38 +323,50 @@ def get_total_irradiance(surface_tilt, surface_azimuth,
     Parameters
     ----------
     surface_tilt : numeric
-        Panel tilt from horizontal.
+        Panel tilt from horizontal.[degree]
     surface_azimuth : numeric
-        Panel azimuth from north.
+        Panel azimuth from north. [degree]
     solar_zenith : numeric
-        Solar zenith angle.
+        Solar zenith angle. [degree]
     solar_azimuth : numeric
-        Solar azimuth angle.
+        Solar azimuth angle. [degree]
     dni : numeric
-        Direct Normal Irradiance
+        Direct Normal Irradiance. [W/m2]
     ghi : numeric
-        Global horizontal irradiance
+        Global horizontal irradiance. [W/m2]
     dhi : numeric
-        Diffuse horizontal irradiance
+        Diffuse horizontal irradiance. [W/m2]
     dni_extra : None or numeric, default None
-        Extraterrestrial direct normal irradiance
+        Extraterrestrial direct normal irradiance. [W/m2]
     airmass : None or numeric, default None
-        Airmass
+        Relative airmass (not adjusted for pressure). [unitless]
     albedo : numeric, default 0.25
-        Surface albedo
+        Surface albedo. [unitless]
     surface_type : None or String, default None
-        Surface type. See grounddiffuse.
+        Surface type. See :py:func:`~pvlib.irradiance.grounddiffuse` for
+        the list of accepted values.
     model : String, default 'isotropic'
-        Irradiance model.
+        Irradiance model. Can be one of 'isotropic', 'klucher', 'haydavies',
+        'reindl', 'king', 'perez'.
     model_perez : String, default 'allsitescomposite1990'
-        Used only if model='perez'. See :py:func:`perez`.
+        Used only if model='perez'. See :py:func:`~pvlib.irradiance.perez`.
 
     Returns
     -------
     total_irrad : OrderedDict or DataFrame
         Contains keys/columns ``'poa_global', 'poa_direct', 'poa_diffuse',
         'poa_sky_diffuse', 'poa_ground_diffuse'``.
+
+    Notes
+    -----
+    Models 'haydavies', 'reindl', or 'perez' require 'dni_extra'. Values can
+    be calculated using :py:func:`~pvlib.irradiance.get_extra_radiation`.
+
+    The 'perez' model requires relative airmass ('airmass') as input. If
+    'airmass' is not provided, it is calculated usign the defaults in
```

Comment:
```suggestion
    ``airmass`` is not provided, it is calculated using the defaults in
```
I can't remember are kwargs supposed to use single backticks or double? I think we have both conventions. Single backticks show up as italics, while double show as syntax highlighting.

Reply:
The intent here is to refer to the kwarg, so syntax highlighting (double backticks).